app/webhook.go

```diff
@@ -145,15 +150,28 @@ func (a *App) TriggerWebhook(payload *model.OutgoingWebhookPayload, hook *model.
 	}
 }
 
-func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType string) (*model.OutgoingWebhookResponse, error) {
-	req, err := http.NewRequest("POST", url, body)
+func (a *App) doOutgoingWebhookRequest(url string, body io.Reader, contentType, secret string) (*model.OutgoingWebhookResponse, error) {
+	bodyBytes, err := ioutil.ReadAll(body)
+	if err != nil {
+		return nil, errors.New("OutgoingWehookRequest payload failed to read.")
+	}
+	bodyReader := bytes.NewReader(bodyBytes)
+
+	req, err := http.NewRequest("POST", url, bodyReader)
 	if err != nil {
 		return nil, err
 	}
 
-	req.Header.Set("Content-Type", contentType)
+	req.Header.Set("Content-Type", contentType) 
```

Comment:
You still have here "unneeded changes" (in this case, an extra space after the line). There are a few on the changes. If you want I can point them all out.